front-end/src/Components/Projects.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+function Projects() {
```

Comment:
lets leave this in components, but change the name to ProjectsList.

because the could be a Project Viewer, or a Project Grader, or a Project Dashboard so we wanna be specific

Reply:
Okay, I just made these changes.